src/engine/QueryExecutionTree.cpp

```diff
@@ -425,7 +425,7 @@ ad_utility::stream_generator::stream_generator QueryExecutionTree::writeTable(
     co_return;
   }
 
-  constexpr char sep = format == TSV ? '\t' : ',';
+  constexpr char sep = format == ExportSubFormat::TSV ? '\t' : ',';
```

Comment:
In this branch (exporting SELECT results) I also would like to see the `static_assert` on `format`
(it is already present for the export of CONSTRUCT results.